src/Adapter/AdapterInterface.php

```diff
@@ -1,85 +1,23 @@
 <?php
 
+declare (strict_types = 1);
```

Comment:
No blank line before this per PSR-12. While it is in draft, we should still adhere to it. https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md

Also, remove all the spaces in this. `declare(strict_types=1)`.